capstone/src/main/java/com/google/sps/data/Room.java

```diff
@@ -98,7 +98,7 @@ public void removeFollower(User follower) {
     }
 
     @SuppressWarnings("unchecked")
-    public List<Key> getAllLessons() { // For Andrew: Your data model showed these as actual Lesson objects. Was that on purpose?
```

Comment:
😄  This kind of comment really shouldnt ever be committed, just send him a ping or something